src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -113,6 +116,16 @@ public static int totalPloidy(final VariantContext vc, final int defaultPloidy)
         return result;
     }
 
+    /**
+     * create a genome location, given a variant context
+     * @param genomeLocParser parser
+     * @param vc the variant context
+     * @return the genomeLoc
+     */
+    public static Locatable getLocation(final VariantContext vc) {
+        return new SimpleInterval(vc.getContig(), vc.getStart(), vc.getEnd());
```

Comment:
This method can be simplified to `return new SimpleInterval(vc);`, since `VariantContext` implements `Locatable` already.

Actually, why do you need this method at all? A `VariantContext` is already a `Locatable` -- no need to create a new one!